subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -70,6 +70,13 @@ Please refer to the <<publishing_maven.adoc#publishing_maven, documentation of t
 The deprecated Play plugins have been removed.
 An external replacement, the link:https://gradle.github.io/playframework[Play Framework plugin], is available from the plugin portal.
 
+==== Removal of experimental JavaScript plugins
+
+The following bundled plugins for experimental JavaScript integration were never announced and are now removed from the distribution:
```

Comment:
:o: You could remove the `never announced` part. It doesn't really help, except for relieving us :)